### DISCUSSION TITLE
CTkToplevel window icon

### DISCUSSION THREAD

**Discussion by cyxigo** [TomSchimansky/CustomTkinter#1383] [Q&A] [answered]
When I try to change the CTkToplevel window icon to my icon, it immediately changes to the standard customtkinter icon, is there a way to fix this?

**Comment by avalon60** [comment, marked as answer]
I think this has been raised as an issue: https://github.com/TomSchimansky/CustomTkinter/issues/1163

**Comment by 0ldManPlaying** [comment]
I used: self.wm_iconbitmap("icon.ico") and did a "_from tkinter import *_"
This worked for me without any issues. 

![image](https://user-images.githubusercontent.com/90275939/232029029-3bf43459-d94d-4d1b-bd6b-8f24a76b1f93.png)
